Content.Server/SimpleStation14/EndOfRoundStats/BloodLost/BloodLostStatSystem.cs

```diff
@@ -0,0 +1,46 @@
+using Content.Server.GameTicking;
+using Content.Shared.GameTicking;
+
+namespace Content.Server.SimpleStation14.EndOfRoundStats.BloodLost;
+
+public sealed class BloodLostStatSystem : EntitySystem
+{
+    float totalBloodLost = 0;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<BloodLostStatEvent>(OnBloodLost);
```

Comment:
you can probably just reference the system and set `totalBloodLost` to whatever in other places, would need the var to be public too

same with all the other stuff

Reply:
I feel doing these with events is better, as it's much more hands off with their respective systems. I can see how this one in particular, as well as shots fired could seem logical to do this way, but at that point it's simply about being consistent.

I find making an event for each one and allowing each system to simply log the information, send it off, and be done is cleaner.

I could, however, see the justification for a broader, more general system that manages *all* of the end round stats.